schemas/generic-serialization.schema.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
If this the schema that all implementations should follow, should it be stored in the organization repository?

Reply:
Eventually, certainly!